### DISCUSSION TITLE
i have copy my serverpod project from window to mac it give me this error on mac when i enter dart bin/main.dart in serverpod server.

### DISCUSSION THREAD

**Discussion by Tehseen890** [serverpod/serverpod#763] [Q&A]
i have copy my serverpod project from window to mac it give me this error.
Unhandled exception:
Null check operator used on a null value
#0 new DatabaseConfig._ (package:serverpod_shared/src/config.dart:142:37)
https://github.com/dart-lang/sdk/issues/1 new ServerpodConfig (package:serverpod_shared/src/config.dart:61:31)
https://github.com/dart-lang/sdk/issues/2 new Serverpod (package:serverpod/src/server/serverpod.dart:242:14)
https://github.com/dart-lang/sdk/issues/3 run (package:pmdccolleges_server/server.dart:14:15)
https://github.com/dart-lang/sdk/issues/4 main (file:///Users/zakaullah/Development/Pmdc_Zaka_final/Pmdc-Serverpod/pmdccolleges_server/bin/main.dart:6:3)
https://github.com/dart-lang/sdk/issues/5 _delayEntrypointInvocation. (dart:isolate-patch/isolate_patch.dart:295:33)
https://github.com/dart-lang/sdk/issues/6 _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:192:26)

**Comment by wstrange** [comment]
Might want to attach a debugger to your process to find out which variable is causing this.

**Comment by vlidholt** [comment]
It looks like Serverpod is failing to read the configuration files for your project. Can you ensure that the `config` directory is the same and that there is a `passwords.yaml` file in it as well. (Passwords are excluded from git if you are managing your project there.)
